blob/blob.go

```diff
@@ -500,7 +480,7 @@ func (b *Bucket) As(i interface{}) bool {
 // ErrorAs converts i to provider-specific types.
 // ErrorAs panics if i is nil or not a pointer.
 // ErrorAs returns false if err == nil.
-// See Bucket.As for more details.
+// See https://godoc.org/gocloud.dev#As for background information.
```

Comment:
You updated `ErrorAs` to reference the main godoc, but didn't update all of the other `As` functions in this package. I don't feel strongly about which one to do (although pointing to Bucket.As is shorter, it's an extra level of indirection), but we should be consistent. Search for `Bucket.As` in this file.

Reply:
Updated to consistently point to doc.go#As